FRONTEND/visualElement.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { View, Image, ImageSourcePropType, StyleSheet } from 'react-native';
+import Animated from 'react-native-reanimated';
+import LottieView from 'lottie-react-native';
+
+interface VisualElementProps {
+  imageSource: ImageSourcePropType;
+  lottieSource: any; 
```

Comment:
Why cant we have predefined type for lottie, e.g. we can either get a JSON for Lottie or the url right? We need to do the same